spoutbreeze-agent/src/main/java/org/spoutbreeze/agent/services/AgentQueueListener.java

```diff
@@ -79,4 +101,18 @@ public void startABroadcastSession(final BroadcastMessage broadcastMessage) {
 
         broadcastRepository.save(broadcast);
     }
+
+    /**
+     * update the broadcast status
+     * @param broadcastMessage the broadcast message
+     */
+    public void updateBroadcastStatus(final BroadcastMessage broadcastMessage) {
+        final Broadcast broadcast =
+                broadcastRepository.findById(broadcastMessage.getId())
+                        .orElseThrow(() -> new RuntimeException("no broadcast with id " + broadcastMessage.getId()));
+
+        broadcast.status = BroadcastStatus.LIVE;
```

Comment:
What's the difference between BroadcastStatus.LIVE and BroadcastStatus.ASSIGNED?

Reply:
The difference should be explained later in the docs.
ASSIGNED comes before LIVE. When the Broadcast gets into the manager; which is distribution system (load-balancer); it assigns a broadcast, now it becomes ASSIGNED. The agent gets that request, the broadcast becomes live when the selenoid session is created.